src/js/components/Box.js

```diff
@@ -175,5 +177,6 @@ Box.defaultProps = {
   direction: 'column',
   pad: 'none',
   tag: 'div',
+  texturePosition: 'center top',
```

Comment:
This feels like a bit too much CSS leaking through into the properties. Instead, would it work to always set "no-repeat center center"?